Makefile

```diff
@@ -3,4 +3,5 @@ mass_push:
 
 mass_publish:
 	cd lambda_bundle && mass bundle lint
+	cd lambda_bundle && mass bundle build
```

Comment:
`mass bundle publish` has `mass bundle build` baked in. So it's not needed here.